### PR TITLE
Card view

### PR DESCRIPTION
**Pull request by Avashist1998**
The card view feature has been completed and the beach can be resolved.

### CODE REVIEW

File: ui/src/components/SubPlayPage.tsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
+import React, { useState } from "react";
+import DrawingCanvas from "./DrawingCanvas";
+
+const SubPlayPage = ({number_truth = "0"}) => {   
+    const [canvasURI, setCanvasURI] = useState("")  
+    const [number_result, SetNumber] = useState(0);
```
Comment (hluong0912) [inline ui/src/components/SubPlayPage.tsx:6]: I think it's better to have this const [numberResult, SetNumberResult] = useState(0);

File: ui/src/components/SubPlayPage.tsx [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    const [score_result, SetScore] = useState(0.99);
```
Comment (hluong0912) [inline ui/src/components/SubPlayPage.tsx:7]: I think it's better to have this const [scoreResult, SetScoreResult] = useState(0);
just a consistent format among the whole project

File: ui/src/pages/Play.tsx [hunk 3]
```diff
@@ -1,42 +1,35 @@
...
+
 
     return (
         <div className="Play">
-            <div className='container'>
-            <DrawingCanvas setImageURI={setCanvasURI}/>
-                <img className="signature-thumbnail" width="250px" 
-                src ={canvasURI} 
-                alt="Signature thumbnail"></img>
-                <button className="submitButton" onClick={test}>Submit</button>
```
Comment (hluong0912) [inline ui/src/pages/Play.tsx:35]: className="submit-button"

**Review by hluong0912 — APPROVED** [review]

File: ui/src/components/SubPlayPage.tsx (hunk 2 above)
Comment (Avashist1998) [reply ui/src/components/SubPlayPage.tsx:7]: I can add those changes after the merge

File: ui/src/pages/Play.tsx (hunk 3 above)
Comment (Avashist1998) [reply ui/src/pages/Play.tsx:35]: the code is no longer in use, I will update it appropriately.

**Pull request merged**